scripts/vsce-bundled-extension.ts

```diff
@@ -0,0 +1,118 @@
+import {
```

Comment:
Would love a lunch and learn to step through each of these steps and understand better what's going on/why

Reply:
100%